src/mozmlops/templates/README.md

```diff
@@ -78,3 +78,81 @@ We know this workaround fails to account for scheduling and scripts: we're worki
 > [!NOTE]  
 > We have changed `offline` here to false: hat means we _do_ want our flow to integrate with Weights and Biases!
 
+## Running Inference Servers in Production
+
+Deploying your Ray Serve app to production requires changes in 3 different repositories: “application repo“, “function specific repo” and “deployment repo“ (please see [glossary](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/785514640/Deploy+inference+servers+to+production+GKE+using+Ray+Serve#Glossary) for details on these repositories).
+
+### Steps in “application repo“
+
+> [!NOTE]
+> We will use the [template Ray Serve app](./template_ray_serve.py) in this repository to demonstrate the command usage in this section. The [original code](https://docs.ray.io/en/latest/serve/develop-and-deploy.html#convert-a-model-into-a-ray-serve-application) for this app comes from the Ray Serve documentation.
+
+#### Test your Ray Serve app locally
+1. Install requirements
+
+    ```sh
+    pip install -r requirements-rayserve.txt
+    ```
+2. Run Ray Serve app locally using [serve run](https://docs.ray.io/en/latest/serve/api/index.html#serve-run) command
+
+    ```sh
+    serve run template_ray_serve:translator_app --route-prefix "/translate"
+    ```
+3. In a different terminal, call the locally running service endpoint and check if it returns the expected response
+    ```sh
+    curl -i -d '{"text": "Hello world!"}' -X POST "http://127.0.0.1:8000/translate/" -H "Content-Type: application/json"
+    ```
+4. Stop the Ray Serve app (kill the `serve run` process) after you are done with testing
+
+#### Create Dockerfile for your Ray Serve app
+Creating docker images is the [recommended way](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/785514640/Deploy+inference+servers+to+production+GKE+using+Ray+Serve#Containerization-of-Ray-Serve-application) to deploy Ray Serve apps to production.
+
+Create a [Dockerfile](https://docs.docker.com/reference/dockerfile/) for your Ray Serve app and add it to the "application repo"
+
+> [!IMPORTANT]
+> The "deployment repo" will build docker image using the Dockerfile that you add to your "application repo".
+
+#### Create a Serve config for your Ray Serve app
+The Serve config is the [recommended way](https://docs.ray.io/en/latest/serve/configure-serve-deployment.html#specify-parameters-through-the-serve-config-file) to deploy and update Ray Serve apps in production.
```

Comment:
A link as to _why_ this is the recommended practice would be nice from my perspective :)

Reply:
@chelseatroy I am changing this link to another link where the explanation for _why_ is also included. Thanks for the suggestion :)